test/e2e/main_test.go

```diff
@@ -225,7 +225,7 @@ func TestManualMeteringInstall(t *testing.T) {
 			MeteringConfigManifestFilename: "node-selector-prometheus-importer-disabled.yaml",
 		},
 		{
-			Name:                      "ValidHDFS-ReportDynamicInputData",
+			Name:                      "HDFS-ReportDynamicInputData",
```

Comment:
I still don't get what these top level test names are trying to convey? Like the config'ed environment all the subtests will operate in? If so there is more than just Storage. And what about Dynamic/Static what's that telling me? I'm not saying fix this for this PR, we can percolate on it for awhile.

Reply:
Yeah, the test names can still be improved. Conceptually, I think we've been interested in conveying what storage type we're running metering on top of and what top-level reporting tests we're going to run once metering is installed and reporting a ready status.